### PR TITLE
Fastpath 0.8: Add anomaly and confirmed columns

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Add columns to ease integration with API and Explorer.

### CODE REVIEW

File: af/fastpath/fastpath/core.py [hunk 1]
```diff
@@ -1046,8 +1047,12 @@ def msm_processor(queue):
                     matches = match_fingerprints(measurement)
                 else:
                     matches = []
+                confirmed = bool(len(matches))
```
Comment (hellais) [inline af/fastpath/fastpath/core.py:1050]: IIRC the fingerprints used by the fastpath differ from those inside of the batch pipeline.

Do we maybe want to mention somewhere this and add a TODO to re-align the two?

Comment (FedericoCeratto) [reply af/fastpath/fastpath/core.py:1050]: I'll create a task in the project dashboard.

File: af/oometa/019-fastpath-confirmed.install.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
+-- Add confirmed and anomaly columns to fastpath table
+-- Formatted with pgformatter 3.3
```
Comment (hellais) [inline af/oometa/019-fastpath-confirmed.install.sql:2]: The rollback file here is missing.

File: af/fastpath/fastpath/core.py (hunk 1 above)
Comment (hellais) [reply af/fastpath/fastpath/core.py:1050]: Can you link it in here?

File: af/oometa/019-fastpath-confirmed.install.sql (hunk 2 above)
Comment (FedericoCeratto) [reply af/oometa/019-fastpath-confirmed.install.sql:2]: Added.

File: af/fastpath/fastpath/core.py (hunk 1 above)
Comment (FedericoCeratto) [reply af/fastpath/fastpath/core.py:1050]: Update: as discussed, with #278 we'll  keep the row in the fastpath db table and read from there. This make the re-align less urgent.

**Review by hellais — APPROVED** [review]
LGTM

**Comment by FedericoCeratto** [comment]
Deployed

**Pull request merged**